bpf/http_types.h

```diff
@@ -96,6 +95,8 @@ typedef struct http_info {
     // with other instrumented processes
     pid_info pid;
     tp_info_t tp;
+    u64 extra_id;
```

Comment:
These are now stored here so we can correctly clean-up the server trace information when the HTTP request is finished.